src/labels/CustomLabels.labels

```diff
@@ -3464,7 +3464,7 @@ Once the process is 100% complete, you can safely leave or refresh this page.</v
         <language>en_US</language>
         <protected>false</protected>
         <shortDescription>stgHelpRDAddCampaign</shortDescription>
-        <value>If selected, adds the campaign that you’ve specified for the Recurring Donation to all related (i.e. “child”) opportunities. If this option is not selected, the campaign appears on the Recurring Donation’s first opportunity only.</value>
+        <value>If selected, adds the campaign that youâve specified for the Recurring Donation to all related (i.e. âchildâ) opportunities. If this option is not selected, the campaign appears on the Recurring Donationâs first opportunity only.</value>
     </labels>
```

Comment:
@jlantz looks like the contributor tool is not dealing well with single and double quotes, or Christian's system is doing it.